common/stages/Functions/library

```diff
@@ -342,3 +342,31 @@ get_partition_node()
 
     return 0
 }
+
+#-----------------------------------------------------------
+# Usage: version_ge <v1> <v2>
+# Parse and compare versions formatted as "major.minor.micro".
+# Return 0 if v1 >= v2, else return 1.
+version_ge() {
```

Comment:
much shorter for sure! But pipes, reads from stdin and would still require processing of the output in the end ;). I'll switch to that if you think it is preferable though.